src/config.ts

```diff
@@ -69,6 +69,19 @@ export interface AdvancedOptionConfig {
         inheritDefault?: StatusBarInheritOptionVisibility;
         statusBarLength?: number;
     };
+    packagePreset?: {
+        projectStatusVisibility?: ProjectStatusOptionVisibility;
+        statusBarVisibility?: StatusBarOptionVisibility;
+        inheritDefault?: StatusBarInheritOptionVisibility;
+        statusBarLength?: number;
+    };
+    workflowPreset?: {
+        projectStatusVisibility?: ProjectStatusOptionVisibility;
+        statusBarVisibility?: StatusBarOptionVisibility;
+        inheritDefault?: StatusBarInheritOptionVisibility;
+        statusBarLength?: number;
+    };
+    // No room for package/workflow presets in status bar
```

Comment:
Is this just referring to the defaults? I see `statusBarVisibility` for package and workflow presets.

Reply:
See https://github.com/microsoft/vscode-cmake-tools/pull/3548#discussion_r1467829227, I commented in another location as well, but we should allow status bar visibility, but it should only be in the project status view by default.